ceph.spec.in

```diff
@@ -804,7 +804,6 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %post
```

Comment:
@ktdreyer We are going to have to add systemd code to the `ceph-osd`, `ceph-mon`, and `ceph-mds` packages, too, at some point.